pandas/tseries/offsets.py

```diff
@@ -231,18 +230,25 @@ def __add__(date):
         - microsecond
         - nanosecond
 
+        .
+
     See Also
     --------
-    dateutil.relativedelta.relativedelta
+    dateutil.relativedelta.relativedelta : The relativedelta type is designed
+
+    to be applied to an existing datetime an can replace specific components of
+
+    that datetime, or represents an interval of time.
```

Comment:
to continue with the description of the see also, avoid the blank lines and indent the continued description

Reply:
Thank you for getting back to me, just committed the changes you requested.